src/server/room-controller.js

```diff
@@ -124,6 +144,12 @@ export default class RoomController {
       (req, res) => this.handlePostRoom(req, res)
     );
 
+    router.delete(
+      '/api/v1/rooms/:roomId',
+      [needsPermission(permissions.CREATE_ROOMS), validateParams(deleteRoomParamsSchema)],
```

Comment:
Should we maybe call this permission `CREATE_OR_DELETE_ROOMS`? :no_good:

Reply:
I'm not sure actually why we need here create/edit/view since everyone can do it (as long as they have a user). Why not just MANAGE_ROOMS?

Reply:
Good point, I'd be fine with that (or something like `OWN_ROOMS`)

Reply:
Added a specific DELETE_ROOMS permission.

Reply:
Fixed to have OWN_ROOMS permission. We will add specific rights when we need them for admins (for example to be able to view rooms that they don't own).